language-server/src/features/document-settings.js

```diff
@@ -1,8 +1,19 @@
 import { DidChangeConfigurationNotification } from "vscode-languageserver";
 import { publish } from "../pubsub.js";
+import picomatch from "picomatch";
 
 
 export const isSchema = RegExp.prototype.test.bind(/(?:\.|\/|^)schema\.json$/);
```

Comment:
This one is still being used in `semantic-tokens.js` here.  https://github.com/hyperjump-io/json-schema-language-tools/blob/bd669013885117f5c18405789fbca34194e2ac43/language-server/src/features/semantic-tokens.js#L54

Reply:
`semantic-tokens.js` needs to be updated too! We only want to respond to semantic token requests if the file is a schema just like we only want to validate and send diagnostics if the file is a schema.

Reply:
Yes, the only reason I kept `isSchema`  as it is here is because `isMatchedFile` was returning wrong values earlier because of probably this.

![image](https://github.com/hyperjump-io/json-schema-language-tools/assets/110971977/dfde27e1-8be3-4fc7-b1b6-ea7f25003c97)
https://github.com/micromatch/picomatch/issues/89

Reply:
The `semantic-tokens.js` is also updated to work for the files defined.
![image](https://github.com/hyperjump-io/json-schema-language-tools/assets/110971977/5fa94a9f-7ba8-47b6-845f-dc9503077ab0)